### PR TITLE
feat(visual story editor): initial commit

### PR DESCRIPTION
**Pull request by pheel**
🎉🎉

### CODE REVIEW

File: botfront/imports/ui/components/utils/IntentLabel.jsx [hunk 1]
```diff
@@ -44,7 +46,13 @@ function Intent({
...
             <Popup
                 trigger={(
-                    <Label id='intent' color='purple' data-cy='intent-label' size={size}>
+                    <Label
+                        id='intent'
+                        color={value !== OOS_LABEL ? 'purple' : 'grey'}
+                        basic={value === OOS_LABEL}
+                        data-cy='intent-label'
+                        size={size}
+                    >
```
Comment (MatthieuJnon) [inline botfront/imports/ui/components/utils/IntentLabel.jsx]: Remove the out of scope logic from here

Comment (pheel) [reply botfront/imports/ui/components/utils/IntentLabel.jsx]: We need it to show (pick intent) with different styling

Comment (MatthieuJnon) [reply botfront/imports/ui/components/utils/IntentLabel.jsx]: mmh ok

**Comment by MatthieuJnon** [comment]
- Bring back Spectrum get help button
- correct Slots modal for height

**Pull request merged**